2016-04/april-7.md

```diff
@@ -0,0 +1,129 @@
+## April 7 ([discuss](https://github.com/reactjs/core-notes/pull/2))
+
+### Releasing React 15
+
+#### Shipping Today
+
+* React 15 is shipping! 🎉
+* Docs should be in good shape.
+* Just need to proofread the [blog post]([#6396](https://github.com/facebook/react/pull/6396)) and ship it.
+
+#### Relay Test Failure
+
+* We changed React to use `object-assign` npm [ponyfill](https://ponyfoo.com/articles/polyfills-or-ponyfills) that falls back to native `Object.assign`. ([#6376](https://github.com/facebook/react/pull/6376))
+* This caused a Relay test to fail internally because native `Object.assign` did not preserve key ordering.
+* Test will be fixed to not rely on this.
+* This won’t block React 15 release.
+
+#### React Native Issue
+
+* `react-native init` install `react@latest` which would break when we release React 15.
+* Sorted out with [James Ide](https://twitter.com/ji).
+* Since we communicate with [Exponent](https://twitter.com/exponentjs) frequently, we were able to fix this preemptively.
+* We’ll release 15 but keep 0.14 `latest` on npm for a while (hopefully a few hours) until React Native gets a fix.
+
+#### Branch Planning
+
+* At a high level, things will be slightly different than they’ve been historically because we now have actual minor versions.
+* We’ll need to maintain both a patch release and a minor release going forward, so we need to figure out how many releases we’re going to support officially.
+* This was also a common question from the community (what is our official story around support of past releases?).
+* Node does labels on every request, we’ll need to figure out what our strategy will be so we cherry-pick all the changes correctly.
+* [Paul](https://twitter.com/zpao) will study how Node does it and come up with a plan.
+
+#### Rolling Changelog
+
+* Let’s maintain a continuously updated changelog so we don’t have to spend hours (sometimes days!) writing release blog entries.
+* We need to figure out how to where to put “unreleased” entries, how to handle merge conflicts, etc.
```

Comment:
There are some good ideas in [Keep a Changelog](http://keepachangelog.com/), especially regarding where unreleased entries go, and also arguing that changelog entries should be written by humans for humans rather than harvested from commit messages.